docs/Examples.md

```diff
@@ -729,3 +738,78 @@ spec:
+
+3. Create a kubernetes secret object whose name is of the format splunk-<namespace_retrieved_above>-secret using the command below. In this example we are creating the global kubernetes secret object with default administrator password and pass4symmkey configured.  
+`kubectl create secret generic splunk-<namespace_retrieved_above>-secret --from-literal='password=<admin_password_value>' --from-literal='pass4symmkey=<pass4symmkey_value>'`
+
+### Reading global kubernetes secret object
+
+Once created, all secret tokens in the secret object are base64 encoded. To read the global kubernetes secret object you can use the following command:
+
+`kubectl get secret splunk-default-secret -o yaml`
+
+A sample global kubernetes secret object with base64 encoded values looks like:
+
+```
+kubectl get secret splunk-default-secret -o yaml
+apiVersion: v1
+data:
+  hec_token: RUJFQTE4OTMtMDI4My03RkMzLThEQTAtQ0I1RTFGQzgzMzc1
+  idxc_secret: VUY5dWpHU1I4ZmpoZlJKaWNNT2VMSUNY
+  pass4SymmKey: dkFjelZSUzJjZzFWOHZPaVRGZk9hSnYy
+  password: OHFqcnV5WFhHRFJXU1hveDdZMzY5MGRs
+  shc_secret: ZEdHWG5Ob2dzTDhWNHlocDFiYWpiclo1
+kind: Secret
+metadata:
+  creationTimestamp: "2020-10-07T19:42:07Z"
+  name: splunk-default-secret
+  namespace: default
+  ownerReferences:
+  - apiVersion: enterprise.splunk.com/v1alpha3
+    controller: false
+    kind: SearchHeadCluster
+    name: example-shc
+    uid: f7264daf-4a3e-4b44-adb7-af52f45b45fe
+  resourceVersion: "11433590"
+  selfLink: /api/v1/namespaces/default/secrets/splunk-default-secret
+  uid: d6c9a59c-1acf-4482-9990-cdb0eed56e87
+type: Opaque
+```
+
+The kubectl command line tool can be used to decode the splunk secret tokens with the following command:
+
+`kubectl get secret splunk-<namespace_retrieved_above>-secret -o go-template=' {{range $k,$v := .data}}{{printf "%s: " $k}}{{if not $v}}{{$v}}{{else}}{{$v | base64decode}}{{end}}{{"\n"}}{{end}}'`
+
+A sample global kubernetes secret object with tokens decoded looks like:
+
+```
+hec_token: EBEA1893-0283-7FC3-8DA0-CB5E1FC83375
+idxc_secret: UF9ujGSR8fjhfRJicMOeLICX
+pass4SymmKey: vAczVRS2cg1V8vOiTFfOaJv2
+password: 8qjruyXXGDRWSXox7Y3690dl
+shc_secret: dGGXnNogsL8V4yhp1bajbrZ5
+```
+
+### Updating global kubernetes secret object
+
+To update the global kubernetes secret object you can use the kubectl command line tool. Here are the steps to do the same.
+
+1. Base64 encode the plain-text value of the secret token using the following command: `echo -n <plain_text_value> | base64`
+2. Obtain the key string for the secret token you are interested in from [here](PasswordManagement.md#splunk-secret-tokens-in-the-global-secret-object). 
+3. Use the encoded value, key string/s to edit the global kubernetes secret object via a patch using the following command: `kubectl patch secret splunk-<namespace_retrieved_above>-secret -p='{"data":{"<key_string_for_secret_token>": "<encoded_value>"}}' -v=1`
```

Comment:
Just for my understanding, what is -v1=1 at the end?

Reply:
Verbosity of logging in kubectl indicated [here](https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-output-verbosity-and-debugging)

Reply:
Done